pkg/vm/strategy.go

```diff
@@ -0,0 +1,104 @@
+package vm
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/firecracker-microvm/firecracker-go-sdk"
+	"github.com/pkg/errors"
+)
+
+// TODO: this is a dead code file. Was used when we use the firecracker-go-sdk to start
```

Comment:
You want to keep this file in case they fix the bug we have with the sdk and we change to use it again ?

Reply:
That's was the plan yes, but we can always restore it, what do u think?

Reply:
I'm ok with that.